tests/suites/test_suite_cipher.blowfish.data

```diff
@@ -2,63 +2,63 @@ BLOWFISH CBC Decrypt empty buffer
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_CIPHER_PADDING_PKCS7
 dec_empty_buf:MBEDTLS_CIPHER_BLOWFISH_CBC:0:0
 
-BLOWFISH Encrypt and decrypt 0 bytes
+BLOWFISH Encrypt and decrypt 0 bytes [#1]
```

Comment:
I would rather opt for having precise test names rather than numbers, so:

- `BLOWFISH-CBC Encrypt [...]`;
- `BLOWFISH-CFB6 Encrypt [...]`;
- `BLOWFISH-CTR Encrypt [...]`;

Reply:
The same applies to CAMELLIA tests.

Reply:
Rejected as out of scope.